### PR TITLE
DartPad export - server enabled

### PR DESCRIPTION
**Pull request by Georgehe4**
Export capabilities enabled - server located at http://export-api.dart-services.appspot.com.
@devoncarew @lukechurch

### CODE REVIEW

**Comment by Georgehe4** [comment]
@lukechurch ready for review.

File: lib/playground.dart [hunk 1]
```diff
@@ -209,7 +212,17 @@ class Playground implements GistContainer, GistController {
       }
     } else {
       editableGist.setBackingGist(createSampleGist());
-    }
+    }/*else if (url.hasQuery &&
```
Comment (lukechurch) [inline lib/playground.dart]: Is there a reason these comments need to remain here?

File: lib/services/common.dart [hunk 2]
```diff
@@ -5,7 +5,7 @@
 library dart_pad.common;
 
 final String serverURL = 'https://dart-services.appspot.com/';
-//final String serverURL = 'http://127.0.0.1:8080/';
+//final String serverURL = 'http://export-api.dart-services.appspot.com/';
```
Comment (lukechurch) [inline lib/services/common.dart]: Lets set this back to the old comment - which is more useful for local testing

File: lib/embed/embed.dart [hunk 3]
```diff
@@ -281,9 +282,15 @@ class PlaygroundMobile {
   }
 
   void _export() {
-    window.open(
+    /*window.open(
         "/index.html?dart=${Uri.encodeQueryComponent(context.dartSource)}&html=${Uri.encodeQueryComponent(context.htmlSource)}&css=${Uri.encodeQueryComponent(context.cssSource)}",
```
Comment (lukechurch) [inline lib/embed/embed.dart]: Long line, could you trim to 80 chars?

File: .gitignore [hunk 4]
```diff
@@ -12,6 +12,10 @@ pubspec.lock
 
 .settings/
 
+#Or intelliJ
```
Comment (lukechurch) [inline .gitignore]: nit: # Or intelliJ

**Comment by lukechurch** [comment]
@Georgehe4 Please address these minor comments, then lgtm.

Please could you also update the title of the PR to be a little more explanatory - e.g. "Enable pad export"

**Comment by Georgehe4** [comment]
Merging.

**Pull request merged**